packages/slider/stories/slider.stories.ts

```diff
@@ -97,12 +143,122 @@ export const Disabled = (): TemplateResult => {
+            }
+        }
+    };
+    return html`
+        <div style="width: 500px; margin: 12px 20px;">
+            <sp-slider
+                step="0.5"
+                min="0"
+                max="20"
+                label="Intensity"
+                @input=${handleEvent}
+                @change=${handleEvent}
+                ...=${spreadProps(args)}
+            >
+                <sp-slider-handle value="5"></sp-slider-handle>
+            </sp-slider>
+        </div>
+    `;
+};
+
+export const TwoHandles = (args: StoryArgs): TemplateResult => {
+    const handleEvent = (event: Event): void => {
+        const target = event.target as SliderHandle;
+        if (target.value != null) {
+            if (typeof target.value === 'object') {
+                action(event.type)(target.value);
+            } else {
+                action(event.type)(`${target.name}: ${target.value}`);
+            }
+        }
+    };
+    return html`
+        <div style="width: 500px; margin: 12px 20px;">
+            <sp-slider
+                value="5"
+                step="1"
+                min="0"
+                max="255"
+                label="Output Levels"
+                @input=${handleEvent}
+                @change=${handleEvent}
+                ...=${spreadProps(args)}
+            >
+                <sp-slider-handle name="min" value="5"></sp-slider-handle>
+                <sp-slider-handle name="max" value="250"></sp-slider-handle>
+            </sp-slider>
+        </div>
+    `;
+};
+TwoHandles.args = {
+    variant: 'range',
+    tickStep: 10,
+};
+
+export const ThreeHandlesOrdered = (args: StoryArgs): TemplateResult => {
+    const handleEvent = (event: Event): void => {
+        const target = event.target as SliderHandle;
+        if (target.value != null) {
+            if (typeof target.value === 'object') {
+                action(event.type)(target.value);
+            } else {
+                action(event.type)(`${target.name}: ${target.value}`);
+            }
+        }
+    };
+    return html`
+        <div style="width: 500px; margin: 12px 20px;">
+            <sp-slider
+                value="5"
+                step="1"
+                min="0"
+                max="255"
+                label="Output Levels"
+                @input=${handleEvent}
+                @change=${handleEvent}
+                ...=${spreadProps(args)}
+            >
+                <sp-slider-handle
+                    name="low"
+                    value="5"
+                    max="next"
+                ></sp-slider-handle>
+                <sp-slider-handle
+                    name="mid"
+                    value="100"
+                    min="previous"
+                    max="next"
+                ></sp-slider-handle>
+                <sp-slider-handle
+                    name="high"
+                    value="250"
+                    min="previous"
+                ></sp-slider-handle>
```

Comment:
I'd love to see `handles` as a Storybook arg here, so that they could be dynamic to the story, but having all three versions available for the VRT is the most important part here, IMO.